### ISSUE TITLE
Control Surface bug or lack of feature?

### ISSUE THREAD

**Issue by djiamnot** [ossia/score#1402]
I just discovered the Control Surface recently. I dropped my OSC device on it and went _whoa!_, thinking that I could immediately control all or chosen OSC parameters. The disappointment came when I realized that I have to manually connect each control to its corresponding OSC address. This seems somewhat counterintuitive, so I think I simply discovered a bug. I observed this in Score 3.0.8 but it's also present in 3.0.9.

**Comment by jcelerier** [comment]
Definitely a bug, normally you should just drop them as you expect and it takes the addresses automatically.. lemme check

**Comment by jcelerier** [comment]
hmmm seems to work here:

https://user-images.githubusercontent.com/2772730/169685507-9ae28182-f3c6-4cd5-81f6-4197e728639a.mp4

**Comment by djiamnot** [comment]
hmmm... yeah, you're right, it works for me, too :shrug: 
Ok, so now, that I've played with it some more, I think that I must have tried to add controls while the timeline was running. I see now, that you have to stop the transport, add control, restart the transport.
Another point that threw me off is the fact that once you've dropped your controls, the addresses associated with them are empty:
![image](https://user-images.githubusercontent.com/777400/169715079-b1a00ffa-76b9-483b-8c4b-efe52fa2165d.png)

**Comment by jcelerier** [comment]
Okay, thanks, indeed I just checked and the live updates aren't being taken into account... fixing that

**Comment by jcelerier** [comment]
fixed. For the addresses not being shown I think it's more of an UI question: since they are controls, they can themselves be controlled through other addresses (in a "mini mapping" fashion if you want). So i guess what remains here is to improve the UI clarity

**Comment by djiamnot** [comment]
Yeah, this was my case, UI vs my expectations. At first, I did not pay much attention because the controls in the info panel are folded by default. When I noticed that there was no effect on the thing I was trying to control, I unfolded and found an empty address. So I dropped the address from the device, which gave me the result that I was expecting: I was controlling the thing. 
Thanks for the quick reaction!
